### DISCUSSION TITLE
Huawei P8 Lite (2015) - the write timed out after xloader

### DISCUSSION THREAD

**Discussion by mjarosie** [kitsuned/PotatoNV#148] [Q&A]
Hi, I managed to connect my device via testpoint and it's being displayed in PotatoNV UI. When I start the bootloader unlock process, I'm getting the following error:

```
PotatoNV v2.2.1
User manual: https://kutt.it/pnv-en
Verifying images...
Uploading hisi620...
- xloader
The write timed out.
```

The "timeout" error happens immediately after `xloader` step gets displayed. Could you give any hints on how I can investigate to find out what's wrong?

When I click "Start!" button again, I'm getting:

```
PotatoNV v2.2.1
User manual: https://kutt.it/pnv-en
Verifying images...
The requested resource is in use.
```

**Comment by MLS-22** [comment]
any news?

**Comment by rock3tsprocket** [comment]
did you fix this?
